scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -255,20 +274,23 @@ task GetBQTableLastModifiedDatetime {
     # ------------------------------------------------
     # try to get the last modified date for the table in question; fail if something comes back from BigQuwey
     # that isn't in the right format (e.g. an error)
-    command {
+    command <<<
         set -e
         if [ ~{has_service_account_file} = 'true' ]; then
             gcloud auth activate-service-account --key-file='~{service_account_json}'
         fi
 
-        LASTMODIFIED=$(bq show --location=US --format=json ~{dataset_table} | python3 -c "import sys, json; print(json.load(sys.stdin)['lastModifiedTime']);")
+        # bq needs the project name to be separate by a colon
+        DATASET_TABLE_COLON=$(echo ~{dataset_table} | sed 's/\./:/')
```

Comment:
I have a branch where I fixed this a different way. by just taking the table name prefix and using the dataset.tablename. I also had an issue where the show command was return a bunch of text prepended to the result which i fixed by creating a .bigqueryrc. I will have a PR for this soon and tag you to review it.